pastas/stressmodels.py

```diff
@@ -694,7 +693,7 @@ def __init__(
             isinstance(rfunc, HantushWellModel) or issubclass(rfunc, HantushWellModel)
```

Comment:
Same comment here as for the issubclass check on line 1424 @davidbrakenhoff mentioned. I believe only the isinstance check is enough. `issubclass(class, classinfo)` returns True if a class is a subclass of classinfo. The subclass here would be RfuncBase for classinfo HantushWellModel (and Exponential). And we don't need to check for RfuncBase right?

Reply:
Agreed, removed.